src/bloqade/squin/op/__init__.py

```diff
@@ -8,19 +8,27 @@
 
 
 @_wraps(stmts.Kron)
-def kron(lhs: types.Op, rhs: types.Op, *, is_unitary: bool = False) -> types.Op: ...
+def kron(lhs: types.Op, rhs: types.Op) -> types.Op: ...
+
+
+@_wraps(stmts.Mult)
+def mult(lhs: types.Op, rhs: types.Op) -> types.Op: ...
+
+
+@_wraps(stmts.Scale)
+def scale(op: types.Op, factor: complex) -> types.Op: ...
 
 
 @_wraps(stmts.Adjoint)
-def adjoint(op: types.Op, *, is_unitary: bool = False) -> types.Op: ...
+def adjoint(op: types.Op) -> types.Op: ...
 
 
 @_wraps(stmts.Control)
-def control(op: types.Op, *, n_controls: int, is_unitary: bool = False) -> types.Op: ...
+def control(op: types.Op, *, n_controls: int) -> types.Op: ...
```

Comment:
Remove wrappers later when syntax sugar is implemented.

Reply:
Just out of curiosity: why? What's the hurt in keeping the wrapper?

Reply:
what syntax sugar are we proposing here for control?